geometry/rotation_body.hpp

```diff
@@ -110,6 +116,64 @@ Rotation<FromFrame, ToFrame>::Rotation(
 }
 
 template<typename FromFrame, typename ToFrame>
+template<typename Scalar, typename F, typename T, typename>
+Rotation<FromFrame, ToFrame>::Rotation(Angle const& angle,
+                                       Bivector<Scalar, FromFrame> const& axis,
+                                       DefinesFrame<ToFrame> tag)
+    : Rotation(AngleAxis(-angle, Normalize(axis).coordinates())) {}
+
+template<typename FromFrame, typename ToFrame>
+template<typename Scalar, typename F, typename T, typename, typename>
+Rotation<FromFrame, ToFrame>::Rotation(Angle const& angle,
+                                       Bivector<Scalar, ToFrame> const& axis,
+                                       DefinesFrame<FromFrame> tag)
+    : Rotation(AngleAxis(angle, Normalize(axis).coordinates())) {}
+
+template<typename FromFrame, typename ToFrame>
+template<typename F, typename T, typename>
+Rotation<FromFrame, ToFrame>::Rotation(
+    Angle const& α,
+    Angle const& β,
+    Angle const& γ,
+    EulerAngles const axes,
+    DefinesFrame<ToFrame> tag)
+    : Rotation(Rotation<ToFrame, FromFrame>(α, β, γ, axes, tag).Inverse()) {}
+
+template<typename FromFrame, typename ToFrame>
+template<typename F, typename T, typename, typename>
+Rotation<FromFrame, ToFrame>::Rotation(
+    Angle const& α,
+    Angle const& β,
+    Angle const& γ,
+    EulerAngles const axes,
+    DefinesFrame<FromFrame> tag)
+    : Rotation(AngleAxis(α, (static_cast<int>(axes) >> (2 * 2)) & 0b11) *
```

Comment:
I'm not sure `binary_coded_ternary_digit_width` and `binary_coded_ternary_least_significant_digit_bitmask` would add to clarity; this reminds me of https://lkml.org/lkml/2016/8/2/1589.
Would a function (templated on the enum class) for `(static_cast<int>(axes) >> (2 * digit)) & 0b11` be better? Maybe `BinaryCodedTernaryDigit(k, axes)`?